intelmq/bots/outputs/cif3/output.py

```diff
@@ -0,0 +1,233 @@
+"""Connect to a CIFv3 instance and add indicator(s).
+
+SPDX-License-Identifier: AGPL-3.0-or-later
+SPDX-FileCopyrightText: 2022 REN-ISAC
+
+A shortened copy of this documentation is kept at `docs/user/bots.rst`, please
+keep it current, when changing something.
+
+Parameters:
+  - add_feed_provider_as_tag: bool, use false when in doubt
+  - cif3_additional_tags: list of tags to set on submitted indicator(s)
+  - cif3_feed_confidence: float, used when mapping a feed's confidence fails or 
+        if static confidence param is true
+  - cif3_static_confidence: bool (use false when in doubt)
+  - cif3_token: str, API key for accessing CIF
+  - cif3_url: str, URL of the CIFv3 instance
+  - fireball: int, used to batch events before submitting to a CIFv3 instance
+        (default is 500 per batch, use 0 to disable batch and send each event as received)
+  - http_verify_cert: bool, used to tell whether the CIFv3 instance cert should be verified
+        (default true, but can be set to false if using a local test instance)
+
+Example (of some parameters in JSON)::
```

Comment:
```suggestion
Example (of some parameters in JSON):
```
or is it intentional?